docs/csharp/tour-of-csharp/tutorials/hello-world.md

```diff
@@ -1,134 +1,184 @@
 ---
-title: Hello World - Introductory interactive tutorial
-description: In this tutorial, you use your browser to learn C# interactively. You write C# code and see the results of compiling and running your code directly in the browser.
-ms.date: 03/05/2025
+title: Hello World - Introductory tutorial
+description: In this tutorial, you create your first C# apps. You write C# code and learn basic structure and types in C#.
+ms.date: 12/10/2025
+# customer intent: As an aspiring developer, I want to learn C#.
 ---
-# Introduction to C# - interactive tutorial
+# Tutorial: Explore the C# language
 
-This tutorial teaches you C# interactively, using your browser to write C# and see the results of compiling and running your code. It contains a series of lessons that begin with a "Hello World" program. These lessons teach you the fundamentals of the C# language.
+This tutorial teaches you C#. You write your first C# program and see the results of compiling and running your code. It contains a series of lessons that begin with a "Hello World" program. These lessons teach you the fundamentals of the C# language.
 
-> [!TIP]
+In this tutorial, you:
+
+> [!div class="checklist"]
 >
-> When a code snippet block includes the "Run" button, that button opens the interactive window, or replaces the existing code in the interactive window. When the snippet doesn't include a "Run" button, you can copy the code and add it to the current interactive window.
+> * Launch a GitHub Codespace with a C# development environment.
+> * Create your first C# app.
+> * Create and use variables to store text data.
+> * Use .NET APIs with text data.
+
+To use Codespaces, you need a GitHub account. If you don't already have one, you can create a free account at [GitHub.com](https://github.com).
```

Comment:
This can either be removed or combined with the first prerequisite.

Reply:
Done across all tutorials.